src/components/schema/_schema_item.scss

```diff
@@ -0,0 +1,79 @@
+/*!
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+.ouiSchemaItem {
+  display: flex;
+  // sass-lint:disable-block no-misspelled-properties
+  gap: $ouiSizeS;
```

Comment:
I assume `gap` was introduced after sass-lint was abandoned?

Reply:
This one surprised me, but gap's been around for a while and is well supported across browsers. I didnt have an option to update sass-lint to recognize this so i had to escape this. I'm open to alternatives though :)